cmd/command/edge/image.go

```diff
@@ -140,11 +141,28 @@ func (p *Plural) handleEdgeImage(c *cli.Context) error {
 		return err
 	}
 
+	if imagePushURL != "" {
+		dockerfile := "FROM scratch\nWORKDIR /build\nCOPY kairos.img /build"
+		dockerfilePath := filepath.Join(buildDirPath, "Dockerfile")
```

Comment:
`buildDirPath` is cleaned up elsewhere I'm guessing?

Reply:
yes